intl/msg_hash_us.h

```diff
@@ -2354,6 +2482,10 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_AUDIO_RATE_CONTROL_DELTA,
    "Helps smooth out imperfections in timing when synchronizing audio and video. Be aware that if disabled, proper synchronization is nearly impossible to obtain."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_AUDIO_RATE_CONTROL_DELTA,
+   "Audio rate control.\nSetting this to 0 disables rate control. Any other value controls audio rate control delta.\nDefines how much input rate can be adjusted dynamically. Input rate is defined as:\ninput rate * (1.0 +/- (rate control delta))"
```

Comment:
Possible bad inheritance here: that first line seems to have no reason of being there.

Reply:
Noted. Several of these help texts did use the menu item name as the first sentence. It makes some sense, because popup help will actually be shown on top of the item so it won't be visible while reading the help. But probably better to unify and remove it everywhere.